csrc/scheduler/matmul.cpp

```diff
@@ -995,9 +1021,24 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
   acr->axis(-1)->parallelize(ParallelType::Vectorize);
   bcr->axis(-1)->parallelize(ParallelType::Vectorize);
 
-  //  0  1  2  3  4   5   6   7  8  9  10 11
-  // [B Mo No Ko Kwo Mwo Nwo Mw Nw (Mi Ni Ki)]
-  if (num_batch_dims != 0) {
+  // Parallelization strategy:
+  //  with splitk:
+  // nbatch +   1   2   3   4    5    6    7    8     9    10    11   12
+  //      -13 -12 -11 -10  -9   -8   -7   -6   -5    -4    -3    -2   -1
+  // [..., Mo, No, Kf, Ko, Kw, Mwo, Nwo, Mwi, Nwi, MNi1, MNi2, MNi3,  Ki]
+  //  (iS) iBx iBy rBz rS  rS  iTz  iTy   iS   iS  iMMA   iTx  iMMA rMMA
```

Comment:
A side question - what is prefix `i` and `r` in this comment?

I think it would be good to add description in this comment to make it clear for readers.

Reply:
Thanks. I will definitely add a comment. These indicate whether the axis is an "iteration" or "reduction" axis; I put these here to make it clear which axes we are reducing, particularly the addition of the grid reduction indicated by `rBz`.